README.md

```
@@ -743,6 +743,166 @@ For specialized setups, you can extend the language mapping:
+(setq lsp-proxy-remote-deploy-mode 'manual)
+```
+
+### Binary Deployment
+
+When the binary check fails (missing or version mismatch on both the global command and the deploy path), lsp-proxy behaves according to `lsp-proxy-remote-deploy-mode`.
+
+#### `manual` (default)
+
+lsp-proxy prints a hint in the minibuffer and waits for you to act:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)